### PR TITLE
Improve several log statements, split resource proof messages.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
@afck: no appropriate reviewer found, use r? to override

File: src/states/node.rs [hunk 1]
```diff
@@ -1940,7 +1940,7 @@ impl Node {
         } else if old_prefix.bit_count() > new_prefix.bit_count() {
             result.add_event(Event::SectionMerge(new_prefix));
         }
-        info!("{:?} Update on RoutingTableResponse completed. Prefixes: {:?}",
+        info!("{:?} Update on RoutingTableResponse completed. Section prefixes: {:?}",
```
Comment (maqi) [inline src/states/node.rs]: is this consistent with the script (collecting routing_table) ?

File: src/peer_manager.rs [hunk 2]
```diff
@@ -490,12 +490,12 @@ impl PeerManager {
...
                     }
                 } else {
-                    trace!("{:?} doesn't have peer_id for {:?}",
+                    trace!("Node({:?}) No peer ID with name {:?}",
                            self.routing_table.our_name(),
                            candidate_name);
                 }
             } else {
-                trace!("{:?} doesn't have peer for {:?}",
+                trace!("Node({:?}) No peer with name {:?}",
```
Comment (maqi) [inline src/peer_manager.rs]: we can now print `self` here, right?

Comment (afck) [inline src/peer_manager.rs]: Unfortunately not, because `self` is mutably borrowed in line 484.

File: src/states/node.rs (hunk 1 above)
Comment (afck) [inline src/states/node.rs]: Ah, right, it isn't. Good catch!

File: src/states/node.rs [hunk 3]
```diff
@@ -986,6 +996,22 @@ impl Node {
...
+            let direct_message = DirectMessage::ResourceProofResponse {
+                part_index: part_index,
+                part_count: part_count,
+                proof: part,
+                leading_zero_bytes: leading_zero_bytes,
+            };
+            self.send_direct_message(peer_id, direct_message)?
+        }
         let elapsed = start.elapsed();
         trace!("{:?} created proof data in {}.{:06} seconds.  Min section size: {}, Target size: \
```
Comment (dhardy) [inline src/states/node.rs]: This now gets printed after sending — is that okay?

Comment (Fraser999) [inline src/states/node.rs]: That's a good point.  This is changing in an upcoming PR to only send the first part here, but I think the elapsed time would be better calculated immediately after creating the proof.

Comment (afck) [inline src/states/node.rs]: Yes, it turns out sending is pretty much instantaneous (the calls, I mean: sending is asynchronous).
But this part will change again anyway, with Fraser's next PR.

File: src/states/node.rs [hunk 4]
```diff
@@ -1247,20 +1294,18 @@ impl Node {
...
+                           name,
                            error);
                 }
             }
-            Ok(None) => {
-                debug!("{:?} adding {:?} into routing table.",
-                       self,
-                       public_id.name());
+            Ok(false) => {
+                debug!("{:?} adding {:?} into routing table.", self, name);
```
Comment (Fraser999) [inline src/states/node.rs]: I'm not sure this log statement isn't just spam - we print inside `add_to_routing_table()` too.

Comment (afck) [inline src/states/node.rs]: Let's remove that in the next PR, as discussed.

**Pull request merged**